openmeetings-web/src/main/webapp/css/custom.css

```diff
@@ -1 +1,20 @@
 /* Licensed under the Apache License, Version 2.0 (the "License") http://www.apache.org/licenses/LICENSE-2.0 */
+.room-block .sb-wb .wb-block .tools .om-icon.big.dweet::before {
```

Comment:
`custom.css` is for users
OM CSS should go to correspondent `raw-*.css` files (most probably to `raw-wd.css`)

Reply:
Done